src/menus/menu-item.jsx

```diff
@@ -155,6 +160,21 @@ const MenuItem = React.createClass({
         React.cloneElement(secondaryText, {style: mergedSecondaryTextStyles}) :
         <div style={this.prepareStyles(styles.secondaryText)}>{secondaryText}</div>;
     }
+    let childMenuPopover;
+    if (menuItems) {
+      childMenuPopover = (
+        <Popover
+          anchorOrigin={{horizontal:'right', vertical:'top'}}
+          anchorEl={this.state.anchorEl}
+          open={this.state.open}
+          onRequestClose={this._onRequestClose}>
+          <Menu desktop={desktop} disabled={disabled} style={{position:'relative'}}>
+            {React.Children.map(menuItems, this._cloneMenuItem)}
+          </Menu>
+        </Popover>
+      );
+      other.onTouchTap = this._onClick;
```

Comment:
What do you think about adding onTouchTap to the `PropType` and the documentation?